### DISCUSSION TITLE
Sprint 3 General Feedback

### DISCUSSION THREAD

**Discussion by 19Si** [The-Basement-Dwellers/DIT-A91906#88] [General]
This is a School Assessment, we would appreciate the feedback but please don't give improved code or any code as it will invalidate the assessment.

This is for the Third and Last? Release.

the general feedback we would like to know:

- What were your last impressions
- How did the controls feel
- How did the enemy feel
- How did the overall game atmosphere feel
- What were your impressions on the overall game

Thanks for checking out our project! :shipit:
[Third Release Link can be viewed from here](https://github.com/The-Basement-Dwellers/DIT-A91906/releases/tag/sprint-3)

**Comment by Monza-Royale** [comment]
the controls felt great,
sometimes the ai didn't see you so you could snipe them through doors, on the whole they tracked better than some of the AAA games I've played in the last month so not complaining. 
overall the game was immersive and fun to play with no game breaking glitches and you have to search for bugs but none that I encountered were problematic

**Comment by Nuddley** [comment]
Movement feels good, dash neesd wokr i went through a wall and my speedrun attempt was ruined. 
Sometimes the attack doesn't register on the enemy,
Ending was a great buildup but not scary.
Needs more levels.

**Comment by kaneviggers** [comment]
Surprisingly solid game ngl,

Found a few bugs with clipping though walls and such, other than that felt real responsive and the post processing and effects were solid, felt like a proper indie game. Could use a few more levels with different situations to use the 2 gameplay mechanics, dash and flash, also a prologue would be pretty cool just story wise, and it's probably a stretch but if you were to take this further, on top of having a campaign it could be smart to have an endless mode with procedurally generating levels with the agents dotted around, increasing the number of them with each level. Sprites and animations were very good as well, and the tutorial was good enough to just teach you the basics

But overall very impressed, if this you took this further and made an actual game out of it, I imagine it doing well!
